_src/static/sass/utilities/_helpers.scss

```diff
@@ -0,0 +1,139 @@
+// ------------------------------
+// edX Pattern Library: Helpers
+
+// About: Sass partial for defining extends and re-usable elements at the Sass compile level.
+
+// #UTILITIES
+// #RESETS
+
+// ----------------------------
+// #UTILITIES
+// ----------------------------
+// visual link
+%fake-link {
```

Comment:
The main use case is if we need to style something that doesn't natively give us the cursor property like this - Its more to cover our butts for never interactions (think sliders or other elements). Happy to remove it if it really gives you the creeps - right now its not used and this will never make it to our compiled CSS as a Sass extend.